nestkernel/recording_backend_mpi.cpp

```diff
@@ -349,15 +351,30 @@ nest::RecordingBackendMPI::get_status( DictionaryDatum& ) const
 }
 
 void
-nest::RecordingBackendMPI::set_status( const DictionaryDatum& )
+nest::RecordingBackendMPI::set_status( const DictionaryDatum& datum )
 {
   // nothing to do
 }
 
 void
 nest::RecordingBackendMPI::get_port( const RecordingDevice* device, std::string* port_name )
 {
-  get_port( device->get_node_id(), device->get_label(), port_name );
+  const std::string& label = device->get_label();
+
+  // The MPI address can be provided by two different means.
+  // a) the address is given via the mpi_address device status
+  // b) the file is provided via a file: {data_path}/{data_prefix}{label}/{node_id}.txt
+
+  // Case a: MPI address is given via device status, use the supplied address
+  if ( !mpi_address_.empty() )
```

Comment:
Shouldn't this be equivalent?
```suggestion
  if ( mpi_address_ )
```
If not, please use this:
```suggestion
  if ( not mpi_address_.empty() )
```

Reply:
Fine by me.

And maybe it is for the better if they take the transition to full Python compatibility slowly in the standardization committee ;-)